dataservice/api/common/schemas.py

```diff
@@ -97,12 +112,13 @@ class RespSchema(Schema):
     return RespSchema
 
 
-def paginated_generator(schema):
+def paginated_generator(resource_name, schema):
     class PaginatedSchema(Schema):
         _status = fields.Dict(example={'message': 'success', 'code': 200})
-        _links = fields.Dict(example={'next': '/participants?after=1519402953',
-                                      'self': '/participants?after=1519402952'
-                                      })
+        _links = fields.Dict(
+            example={'next': '/{}?after=1519402953'.format(resource_name),
```

Comment:
might want to do a `resource_name.replace('_', '-')` here

Reply:
Oh i see, 👍